res/lang/rooms/rooms_en.txt

```diff
@@ -1,44 +1,44 @@
-ROOM_ELEVATORPLAN        = Elevator target planner
-ROOM_NEWSPLANNER         = News planner
+ROOM_ELEVATORPLAN        = Elevator Room Planner
+ROOM_NEWSPLANNER         = News Planner
 ROOM_FINANCES            = Finances
 ROOM_SAFE                = Safe
-ROOM_PROGRAMMEPLANNER    = Programme planner
-ROOM_STATIONMAP          = Station map
-ROOM_ROOMMAP             = Room map
+ROOM_PROGRAMMEPLANNER    = Programme Planner
+ROOM_STATIONMAP          = Station Map
+ROOM_ROOMMAP             = Room Map
 ROOM_MOVIEAGENCY         = Movie Agency
-ROOM_MOVIEAGENCY_OWNER   = J. Steward Jabble
+ROOM_MOVIEAGENCY_OWNER   = Proprietor: J. Steward Jabble
```

Comment:
Not sure if this works globally. If the key is not only used for the room signs...
@GWRon ?

Reply:
The Owner-Part is in my opinion only used in the "building-screen"-room-tooltips - so should not create problems.